_data/team/roles/outreach.yml

```diff
@@ -0,0 +1,42 @@
+role: Outreach
+subroles:
+  - subrole: Workshops and meetings
+    tasks:
+    - Identifying workshop and event opportunities
+    - Organising workshops and events
+    - Gathering and coordinating volunteers
+    - Managing content, presentations and teaching at workshops
+    current_leads:
+      - Jenna Swarthout Goddard
+    current_members:
+      - Micaela Matta
+      - All core devs
```

Comment:
Same comment as in other context: It makes no real sense to list a named coredev and "all coredevs" as this implies that the named person isn't a coredev. In this case  I'd very much suggest to promote @micaela-matta to a lead role (also reflecting the work that she's currently doing).

Reply:
@micaela-matta are you happy with that? Otherwise I would actually just remove "all core devs" here as this table is for highlighting people who make special effort, I guess, and the "all core devs" mostly for the "teaching" part of this.

Reply:
As decided in the last core devs meeting, we won't distinguish leads here so I'll keep this as simply "all core devs".